cpg_workflows/stages/joint_genotyping_qc.py

```diff
@@ -69,53 +69,57 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
 
 
 @stage(required_stages=JointGenotyping)
-class JointVcfHappy(SampleStage):
+class JointVcfHappy(SequencingGroupStage):
     """
     Run Happy to validate validation samples in joint VCF
     """
 
-    def expected_outputs(self, sample: Sample) -> Path | None:
+    def expected_outputs(self, sequencing_group: SequencingGroup) -> Path | None:
         """
         Parsed by MultiQC: '*.summary.csv'
         https://multiqc.info/docs/#hap.py
         """
-        if sample.participant_id not in get_config().get('validation', {}).get(
-            'sample_map', {}
-        ):
+        if sequencing_group.participant_id not in get_config().get(
+            'validation', {}
+        ).get('sample_map', {}):
```

Comment:
As per _gvcf_qc.py_: are we leaving this configuration key as `sample_map`?

Reply:
As above